modules/claims_api/spec/lib/claims_api/local_bgs_proxy_spec.rb

```diff
@@ -12,15 +12,12 @@
   end
 
   expected_instance_methods = {
-    construct_itf_body: %i[options],
     convert_nil_values: %i[options],
     find_by_ssn: %i[ssn],
-    find_intent_to_file_by_ptcpnt_id_itf_type_cd: %i[id type],
     find_poa_by_participant_id: %i[id],
     find_poa_history_by_ptcpnt_id: %i[id],
     find_tracked_items: %i[id],
     healthcheck: %i[endpoint],
-    insert_intent_to_file: %i[options],
```

Comment:
we're going to get rid of the proxy and the proxy spec so we don't have to keep maintaining this. I'll note to add a ticket.